package.json

```diff
@@ -23,6 +23,7 @@
 		"babel-loader": "^7.1.5",
 		"babel-preset-env": "^1.7.0",
 		"babel-preset-metal-jsx": "^0.1.1",
+		"classnames": "^2.2.6",
```

Comment:
hey @interaminense, You can move `classnames` to `dependencies`. It is necessary for the `electron-packager` to mount the build...